namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-court-data-adaptor-prod/resources/route53.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_route53_zone" "laa_crime_apps_team_route53_zone" {
+  name = "court-data-adaptor.service.justice.gov.uk"
+
+  tags = {
+    business-unit          = "Legal Aid Agency"
+    application            = "LAA Court Data Adaptor"
+    is-production          = "true"
+    environment-name       = "production"
+    owner                  = "LAA Crime Apps Team: laa-crime-apps@digital.justice.gov.uk"
+    infrastructure-support = "laa-crime-apps@digital.justice.gov.uk"
+  }
+}
+
+resource "kubernetes_secret" "laa_crime_apps_route53_zone_sec" {
+  metadata {
+    name      = "laa-crime-apps-route53-zone-output"
+    namespace = "laa-court-data-adaptor-uat"
```

Comment:
can you please change the namespace name to `laa-court-data-adaptor-prod`

Reply:
Thanks, updated!